docs/docs/ecommerce-tutorial/index.md

```diff
@@ -313,7 +313,7 @@ Lastly make sure that your `.gitignore` file excludes all of your `.env.*` files
 
 ### Create a component that lists your SKUs
 
-In your components folder add a new `Products` folder. This folder will entail your components that interact with the Stripe SKUs. First you need a components that queries and lists your SKUs:
+In your components folder add a new `Products` folder. This folder will entail of your components that interact with the Stripe SKUs. First, you need a component that queries and lists your SKUs:
```

Comment:
` This folder will entail of your components` I believe this shouldn't be changed `This folder will entail your components` is more correct or at least it reads better.

Reply:
I like "include" more, too